settings.gradle

```diff
@@ -4,4 +4,5 @@ include("processor")
 include("common")
 include("core")
 include("test-utils")
-include("it")
\ No newline at end of file
+include("it")
+// include("benchmark")
```

Comment:
Why is this commented out?

Reply:
I only ran the benchmark locally to get some numbers and since we don't run benchmarks on any server I thought it would be wasted to compile it.

Reply:
Ah okay, maybe just add that as a comment then :)